### ISSUE TITLE
GraphRoutes package

### ISSUE THREAD

**Issue by grabos** [levinzonr/compose-safe-routing#26]
During developing our project in couple developers we recognize that is some bug with generating `{GraphName}GraphRoutes` class. For different developer this class has different package location, so often we have conflict in git.
Avery of us have the same versions of code. For example out main package is: `com.example.app`, and we Create some screen: `Splash`, `Login`, `Register`:
`com.example.app.splash.Splash`
`com.example.app.auth.Login`
`com.example.app.auth.Regster`
For all of this screens we are using `@Route(name = "NAME_OF_SCREEN")`

For me, lib will create classs `MainGraphRoutes` in `com.example.app.splash.MainGraphRoutes`, but for next developer it will be `com.example.app.auth.MainGraphRoutes`.

Is any way to define in which package `MainGraphRoutes` should be generated?

**Comment by levinzonr** [comment]
Hey @grabos :) 
Thanks for submitting the issue :)
Can I ask what version and processor are using (ksp or kapt)? 
The way "main" package name is determined is by taking the first class the processor peeks up and taking its package name, which might cause this issue

Think I can come up with something over the weekends and make a fix for it

**Comment by grabos** [comment]
We are using newest version (2.5.1) and `kapt` but issue is on 2.4.* too. 
Thanks :)

**Comment by levinzonr** [comment]
Hey @grabos 
I've published 2.5.2 where You can specify the default package name using processor arguments, for kapt you can try this
```kapt {
    arguments {
        arg("safeRoute.defaultPackageName", "cz.levinzonr.saferoute.navigation")
    }
}```

Let me know how that works for you :)
